### PR TITLE
add types

### PR DESCRIPTION
**Pull request by Fibii**
Adds TS types for `./packages`

### CODE REVIEW

**Review by leofmds — CHANGES_REQUESTED** [review]

File: packages/types/package.json
[the remark this answers is not in the document]
Comment (seanspeaks) [reply packages/types/package.json:1]: @leofmds Can we just do this ourselves? (and merge in)

[the remark this answers is not in the document]
Comment (leofmds) [reply packages/types/package.json:1]: @seanspeaks Sure, I can do it

**Review by leofmds — APPROVED** [review]

**Comment by seanspeaks** [comment]
:rocket: PR was released in [`@friggframework/api-module-microsoft-sharepoint@0.0.5`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-microsoft-sharepoint%400.0.5), [`@friggframework/types@0.1.1`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/types%400.1.1) :rocket:

**Comment by seanspeaks** [comment]
:rocket: PR was released in [`@friggframework/api-module-frontify@0.0.10`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-frontify%400.0.10) :rocket:

**Pull request merged**